packages/wonder-blocks-cell/src/theme/default.ts

```diff
@@ -77,7 +80,7 @@ export default {
         sizing: {
             height: sizing.size_020,
         },
-        shadow: `inset 0px -1px 0px ${semanticColor.core.border.inverse.default}`,
+        shadow: `inset 0px -1px 0px ${semanticColor.core.border.neutral.subtle}`,
```

Comment:
Also highlighted in the [Chromatic diff](https://www.chromatic.com/test?appId=5e1bf4b385e3fb0020b7073c&id=689f6be2bac67cac4cb11e34): 
> The cell divider was using `border.inverse.default (fadedOffBlack8)`. `fadedOffBlack8` isn't used for a semantic border token so as an alternative, `border.neutral.subtle (fadedOffBlack16)` is used instead!